src/main/java/org/fairdatapoint/service/schema/MetadataSchemaMapper.java

```diff
@@ -231,6 +231,10 @@ public MetadataSchemaVersion toDraft(MetadataSchemaVersion schema) {
                 .definition(schema.getDefinition())
                 .suggestedResourceName(schema.getSuggestedResourceName())
                 .suggestedUrlPrefix(schema.getSuggestedUrlPrefix())
+                .schema(schema.getSchema())
```

Comment:
Would it make sense to use `schema.toBuilder()` instead of `MetadataSchemaVersion.builder()`?
Or is there a specific reason not to?

For example:

```java
        return schema.toBuilder()
                .published(false)  // assuming this needs to be false for a draft
                .state(MetadataSchemaState.DRAFT)
                .type(MetadataSchemaType.CUSTOM)
                .createdAt(Instant.now())  // not sure if these should be new timestamps or copies
                .updatedAt(Instant.now())
                .build();
```

Reply:
That is true... should it be done as part of this PR?

Reply:
> should it be done as part of this PR?

Perhaps only for this method, because it is touched anyway in the current PR, and then open a separate issue to refactor the other methods for consistency?